README.md

```diff
@@ -78,25 +82,83 @@ Run the following steps on your laptop or local workstation:
         cd dockerflow
         mvn package -DskipTests
 
-3. Set up the DOCKERFLOW_HOME environment.
+3.  Set up the DOCKERFLOW_HOME environment.
 
         export DOCKERFLOW_HOME="$(pwd)"
         export PATH="${PATH}":"${DOCKERFLOW_HOME}/bin"
         chmod +x bin/*
 
-4.  Run a sample workflow:
+4.  Set up your local project environmental variables.
```

Comment:
The pattern throughout our docs and cookbooks is to have people fill in their own project and bucket in the commands. This has been following the pattern of existing Google Cloud documentation, but I don't know if there is a policy or just a common convention.

My take on it is that if there are just a few commands to run then it is a good idea to encourage people to read the command and fill in the necessary fields. If I'm setting a variable once so that I don't have to otherwise type it twice, there's not a large win and I might be less inclined to read what I am copy/pasting.

If it is a dozen commands, then it gets tedious and it is probably a good idea to alleviate the friction.